build/images/extract.sh

```diff
@@ -51,6 +51,8 @@ function extract-images() {
     YQ="docker run --rm -i \"$YQ_IMAGE\""
 
     images="$( bash <<EOF
+set -eo pipefail
```

Comment:
This is already set on line 3.

Reply:
Right, but that doesn't cause the inline script to exit

Reply:
My understanding is that a new bash interpreter is created on line 53, and that the original set command on line 3 isn't inherited and therefore has no effect on the script piped into the bash interpreter using the here document construct.